extra/lib.sh

```diff
@@ -125,9 +126,17 @@ function letsencrypt_cert() {
     __mydomain=$__domain
   fi
 
-  /usr/bin/certbot-auto certonly -n --agree-tos --standalone --standalone-supported-challenges tls-sni-01 -m "$__myemail" -d "$__mydomain"
-  sudo ln -s "/etc/letsencrypt/live/$__mydomain/cert.pem" "$1" || true
-  sudo ln -s "/etc/letsencrypt/live/$__mydomain/privkey.pem" "$2" || true
+  if [[ $__docker = true ]]; then
+    echo "#!/bin/bash" >> /root/tmp/certbot.sh
```

Comment:
This is nasty, why don't we have a ```certbot_helper.sh``` already created distributed by our platform, and we either run it or copy it to the ```/tmp``` location?